docs/www/configuration.md

```diff
@@ -12,6 +12,8 @@ The `redpanda` section contains all the runtime configuration, such as the
 cluster member IPs, the node ID, data directory, and so on. The `rpk` section
 contains configuration related to tuning the machine that redpanda will run on.
 
+## Sample configuration
+
 Here’s a sample of the config. The [configuration reference](config-reference.md) shows a more complete list of the configuration options.
```

Comment:
removing config-reference breaks all the links, so you'll need to fix those as well across the www and the docs